internal/pkg/cli/deploy/backend.go

```diff
@@ -91,12 +91,13 @@ func (d *backendSvcDeployer) stackConfiguration(in *StackRuntimeConfiguration) (
 		return nil, err
 	}
 	conf, err := stack.NewBackendService(stack.BackendServiceConfig{
-		App:           d.app,
-		EnvManifest:   d.envConfig,
-		Manifest:      d.backendMft,
-		RawManifest:   d.rawMft,
-		RuntimeConfig: *rc,
-		Addons:        d.addons,
+		App:                d.app,
+		EnvManifest:        d.envConfig,
+		Manifest:           d.backendMft,
+		RawManifest:        d.rawMft,
+		ArtifactBucketName: d.resources.S3Bucket,
```

Comment:
Is bucketName itself is enough to pass down to cfn or do we need S3Key as well?

Reply:
S3 key will computed in the follow-up PR :)